### PR TITLE
feat: start page, hubs page, docs page, misc polish

### LINKED ISSUE #1095: Studio Press Release Blog Post
Create a blog post for the Studio Press Release, the same as we did for the Gartner Press Release. Set me as the author.

Needs to go up chronologically before Marc's blog post.

Here is the press release for Studio - https://www.businesswire.com/news/home/20190820005232/en/Stoplight-Launches-API-Visual-Editor-Aimed-Driving

Here is the way we did it for the Gartner Press Release  - https://stoplight.io/blog/gartner-cool-vendor/

### LINKED ISSUE #1096: Marc's Studio Announcement Blog Post
**Chore summary**
Create blog post, Marc set as the author, for Studio announcement. Different from the Press Release Blog post. Needs to go up chronologically after the Press Release blog post. Content for blog post below, link to Google Doc as well.

https://docs.google.com/document/d/1SdR4Sy5OtRtAHpsWD04dDWayE3hoDPxEsqIAlRBfuSM/edit?pli=1#

-------
 
# Stoplight Studio Enables Your API Design Management at Scale
Subtitle: A desktop or web API design experience connected to your own version control system

API tooling is full of walled gardens, with vendors offering solutions for API reference documentation, editors, mocking, code generation, testing, usually all locked up in their own platforms. This makes integration with your own workflows and build pipelines tricky, and means your API descriptions are somewhere other than where they should be: with your code. Stoplight used to work this way, but we're committed to solving this problem.

Today we are sharing the new [Stoplight Studio](https://stoplight.io/studio/), an API design management experience, built with modern development teams in mind. Previous versions of Stoplight required you to keep your API description documents on our servers, but now Stoplight Studio connects to your existing version control! We analyze repos to find relevant files, and give you a beautiful visual editor to make updates or add new information. Then you can commit the changes back to your Git server (GitHub, BitBucket, GitLab, etc).

You also don’t need to convince your entire team to use Stoplight Studio (though we think they’ll come around). We work within your existing file structure, pulling the latest from your OpenAPI, Markdown, and JSON Schema files, as long as they’re committed to a repo we can access. Got a teammate who loves writing YAML in their favorite text editor? No worries, Stoplight Studio can work with that.

We’ll share a lot more about Stoplight Studio over the coming weeks, but here’s a quick look at what is available today:

- Desktop editor experience with offline support
- Built-in support for [Prism](https://stoplight.io/prism/) mock servers on your own machine
- Powerful YAML/JSON linting from [Spectral](https://stoplight.io/spectral/), automatically included
- New web editor interface interoperable with the desktop Studio
- Smooth design-to-docs process to easily share your API functionality

Stoplight Studio is completely free for individuals. We want everyone to have access to the best API design management tooling without compromises or lock-in. For five years, that’s been my goal, and what we’ve built along the way has helped us deliver on that promise.
## First Visual API Designer for API Descriptions
When I founded Stoplight, description formats were getting in the way of API designers. You could use documentation-focused Swagger (which later became OpenAPI) or modeling-centric RAML. In either case, you had to write the YAML or JSON yourself, which caused a barrier to entry for many would-be adopters.

We built the first visual API designer to enable anyone to describe an API without first picking and learning syntax. You could bring an existing document, or start fresh with a builder that walks you through the elements of an API definition. Model request and response data, declare HTTP methods, and field types. Much of this groundbreaking initial interface remains in Stoplight Studio.

Our first API design tool saved valuable developer time and opened up the process to the whole team. Without memorizing formats, enterprise architects and product managers alike could build APIs 10 times faster than using any non-visual editor.
## True OpenAPI Editor Built on Git
When OpenAPI emerged as the industry choice, we wanted our design editor to fully support the format. In 2018, we built the next version to adopt this standard. While our original editor imported and exported multiple API descriptions, it rebuilt the files. That meant we assumed only the standard OpenAPI and anything else, such as extensions, would be lost when rebuilt. Our second visual editor improved upon this flaw and was completely lossless, always providing a view of your OpenAPI document (including any extensions). It also tackled versioning and customizable documentation.

Our customers build and update many APIs, which brings a versioning challenge. We attempted to solve the problem with a tool well known for branching and merging. Our new editor was built atop a hosted Git server, which allowed for tracking revisions and defining versions. We provided a true OpenAPI editor that could go back and forth between a visual editor and the document itself.

Any team member could use the editor, add comments, make updates, and see all the changes. Our customers love the experience and insight it gives them into their API design and development process. Plus, we gave them the tools to share their API beyond the immediate team.

Every API needs documentation, so we created a flexible docs generator from OpenAPI. Though Stoplight Studio supersedes the editor, this documentation tool remains as [Stoplight Hubs](TBD/hubs). Add your own styles, generate API references, and add additional documentation pages. A consumer’s complete experience of your API includes understanding how it works, what is possible, and how to get started. Hubs help you educate developers, whether external or internal while ensuring that documentation is automatically updated along with your OpenAPI document.
## No Compromise API Design Management
Our API design application had solved many of our customer’s problems, but we inadvertently created yet another walled garden. Our own hosted Git repository became the source of truth. Updates to API versions required following our opinionated workflow. And in order to get the maximum benefit, everyone in your organization needed to use our tool.

With Stoplight Studio, our aim is to become part of your workflow, not require you to conform to ours. We connect to your own Git server and bring the lossless OpenAPI visual editor you’ve come to expect. Changes are saved locally until you choose to push them to your repository. We can also detect external changes so you’re always working with the latest version.

In addition to using OpenAPI documents for API design and documentation, we know they’re useful across the API lifecycle. Our team contributes to many open source projects around API tooling, including some we’ve originated ourselves. Two of these are bundled with Stoplight Studio:

- Spectral, a [flexible JSON linter](https://stoplight.io/blog/spectral-v4/), ensures complete OpenAPI definitions and helps your organization design consistent APIs.
- Prism, an [incredibly fast mock server](https://stoplight.io/blog/prism-v3/), can fake your endpoints and data based on OpenAPI documents.

Stoplight Studio is the first of a broader set of tools from Stoplight to help organizations move design-first with their APIs. Collaborate across teams to get feedback—through mock servers, early documentation, and a visual API editor—before you write code. Then make sense of all your API assets at any stage in the lifecycle, right from your own source.

We invite you to [try Stoplight Studio now](https://stoplight.io/studio/). Both desktop and web versions are free. You can open any GitHub URL inside the editor and begin to explore how we interpret your API files. Prototype new endpoints, generate mock servers, or quickly publish documentation. See how Stoplight Studio can provide API design management at scale. No lock-in, no compromises.

### PR DESCRIPTION
**Pull request by marbemac**
fixes #1096 
fixes #1095

### CODE REVIEW

File: netlify/landings/documentation.yaml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+featureSection:
+  actionBar:
+    ctas:
+      - color: accent
+        href: 'https://next.stoplight.io/'
+        title: Get Started Now
+    enabled: true
+    text: 'Ready to Publish? '
+  description: >-
+    Create API documentation in minutes. Combine OpenAPI Specifications
```
Comment (philsturgeon) [inline netlify/landings/documentation.yaml]: This has the same description as `/docs`. Sorry if I seem like a dog with a bone but why do we have /docs, /documentation and /hubs?
```suggestion
    Create API documentation in minutes. Combine OpenAPI descriptions
```

File: netlify/landings/docs.yaml [hunk 2]
```diff
@@ -1,76 +1,54 @@
...
-    text: 'Ready to Publish? '
+  title: Built for Open Source
   description: >-
-    Create API documentation in minutes. Combine OpenAPI Specifications
-    (formerly known as Swagger) with Markdown to create robust, searchable,
-    internal and external [API documentation](/api-documentation-guide/basics/)
-    for your organization. One-click cloud deployment with Stoplight or download
-    the static assets to host anywhere.
+    Create API documentation in seconds using any git repository. Combine OpenAPI Specifications (formerly known as Swagger) with Markdown to create robust, searchable, [API documentation](/api-documentation-guide/basics/)
+    for any project. One-click publish from [Stoplight Studio](/studio) or add the Stopight CLI in your CI/CD workflow.
```
[the remark this answers is not in the document]
Comment (marbemac) [reply netlify/landings/docs.yaml]: Skip the `/docs` page for the moment, that's the main thing left to do here :).

File: netlify/landings/documentation.yaml [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+relatedTags:
+  - blog-documentation
+hero:
+  bgColor: green
+  image:
+    alt: Documentation
+    src: /images/hubs/hero.png
+  subtitle: >-
+    Create beautiful, customizable, interactive API documentation generated from
+    your OpenAPI Specification
```
Comment (philsturgeon) [inline netlify/landings/documentation.yaml]:
```suggestion
    your OpenAPI descriptions
```

File: netlify/landings/documentation.yaml [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+        assets or use our on-premise solutions behind your firewall to host your
+        documentation from anywhere.
+      image: /images/flexible_deployment.png
+      shortName: Integrations
+      title: 'Simple, Flexible Deployment'
+      titleURL: /documentation/hosted/
+  title: Our OpenAPI Documentation Tool
+meta:
+  description: >-
+    Create API documentation in minutes. Combine OpenAPI Specifications
```
Comment (philsturgeon) [inline netlify/landings/documentation.yaml]:
```suggestion
    Create API documentation in minutes. Combine OpenAPI descriptions
```

File: netlify/landings/documentation.yaml [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+    (formerly known as Swagger) with Markdown to create robust, searchable,
+    internal and external API documentation for your organization. One-click
+    cloud deployment with Stoplight or download the static assets to host
+    anywhere.
+  favicon: /images/mark_light_bg.png
+  image: /images/docs_header.png
+  title: REST API Documentation Generator | Hosted API Documentation
+  twitter:
+    description: >-
+      Create API documentation in minutes. Combine OpenAPI Specifications
```
Comment (philsturgeon) [inline netlify/landings/documentation.yaml]:
```suggestion
      Create API documentation in minutes. Combine OpenAPI descriptions
```

File: netlify/landings/hubs.yaml [hunk 6]
```diff
@@ -75,6 +75,44 @@ featureSection:
...
+      tag: "FREE"
+      buttonText: "Learn more about free docs"
+      href: "/docs"
+      features: ['OpenAPI v2 & v3', 'HTTP Request Maker', 'Unlimited Published Pages']
+
+    - isDark: true
+      icon: ['fad', 'user-friends']
+      title: "Pro"
+      tag: "$179 / month"
+      href: "https://next.stoplight.io"
```
Comment (philsturgeon) [inline netlify/landings/hubs.yaml]: Is Hubs the old one? Is that why we're pointing to Next?

File: netlify/landings/hubs.yaml [hunk 7]
```diff
@@ -1,6 +1,6 @@
 path: /hubs
```
Comment (philsturgeon) [inline netlify/landings/hubs.yaml]: I noticed the image for "Custom Pages, Layouts, and Design" is  and that doesnt relate to customization.

File: netlify/landings/hubs.yaml (hunk 6 above)
Comment (marbemac) [reply netlify/landings/hubs.yaml]: Yeah, until we have custom domains and branding in platform docs, this is basically how we decided to split it up. "Paid / company docs" go to hubs, "free / simple" docs is the newer platform stuff. Not ideal, but it's the best we could come up with - hopefully we're able to merge it all and ditch next by the end of the year.

Comment (philsturgeon) [reply netlify/landings/hubs.yaml]: Ok. Maybe we could have an all hands announcement showing off the new site when its launched, and explaining a few things like:

1. docs vs hubs
2. pricing 
3. timelines of when these things might be resolved

If I'm confused other folks on the team probably are too, and if we are then customers probably will be.

Comment (marbemac) [reply netlify/landings/hubs.yaml]: Yeah, we'll have an all hands second half of next week.

File: netlify/landings/docs.yaml [hunk 8]
```diff
@@ -10,7 +10,7 @@ hero:
     Generate beautiful technical documentation for your git projects
   image:
     alt: Documentation
-    src: /images/docs/landing.png
+    src: /images/docs/studio.png
```
Comment (marbemac) [inline netlify/landings/docs.yaml]: heh, didn't like the landing image?

File: netlify/blog-posts/studio-launch.md [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+
+Previous versions of Stoplight required you to keep your API description documents on our servers, but Stoplight Studio includes first class git support (yes, even in the browser!), which allows you to use your existing version control system. We analyze git repositories to find relevant files, and give you a beautiful visual editor to create your API designs. When ready, you can commit the changes directly to your git repository (GitHub, Bitbucket, GitLab, etc). This makes integration with your own PR workflows and build pipelines easy, and means your API descriptions and docs can now live right alongside your code.
+
+We’ll share a lot more about Stoplight Studio over the coming weeks, but here’s a quick look at what is available today:
+
+- Studio Web, with first class Github support (our enterprise platform supports other VCS providers)
+  - Open any github repository by visiting a URL like `https://stoplight.io/p/studio/gh/{yourGithubOrg}/{yourGithubProject}`
+- Studio Desktop for working offline, with files on your computer
+- Editors for Open API v2 & v3, standalone JSON Schema models, and markdown files.
```
Comment (philsturgeon) [inline netlify/blog-posts/studio-launch.md]:
```suggestion
- Editors for Open API v2 & v3, standalone JSON Schema models, and Markdown files.
```

File: netlify/landings/docs.yaml [hunk 10]
```diff
@@ -1,93 +1,80 @@
...
-        Stoplight offers multiple ways for your API consumers to try out your
-        API. Test Requests within your documentation allows consumers to
-        understand how your API works. Code Examples generated from dozens of
-        languages such as JavaScript, Python, Ruby, Java, cURL, and more allow
-        consumers to copy and paste right into their own code.
-      image: /images/docs_feature_1.png
-      shortName: API Playground
-      title: Interactive API Documentation
-      titleURL: /documentation/beautiful/
+        Automatically generate documentation from your OpenAPI v2 or v3 documents. With a powerful request maker, your users will be able to try out your API directly from the docs. Our JSON schema viewer allows them to easily navigate deeply nested request and response structures.
```
Comment (philsturgeon) [inline netlify/landings/docs.yaml]:
```suggestion
        Automatically generate documentation from your OpenAPI v2 and v3 documents. With a powerful request maker, your users will be able to try out your API directly from the docs. Our JSON Schema viewer allows them to easily navigate deeply nested request and response structures.
```

File: netlify/landings/docs.yaml [hunk 11]
```diff
@@ -1,93 +1,80 @@
...
-        Customize your documentation with Themes, Custom CSS, Pages, Domains,
-        and Versioning. With fully customizable layouts, page design and our
-        Markdown-based Editor, you can document multiple APIs and add rich
-        content and resources alongside API reference documentation for a more
-        complete view of your platform.
-      image: /images/product/docs-theme.png
-      shortName: Theming
-      title: Fully Customizable
-      titleURL: /documentation/helper/
+        Stoplight Docs uses a powerful markdown engine built on top of the CommonMark spec. Enhance your documentation with embedded models, code snippets, call outs and more to give your users the best experience.
```
Comment (philsturgeon) [inline netlify/landings/docs.yaml]:
```suggestion
        Stoplight Docs uses a powerful Markdown engine built on top of the CommonMark spec. Enhance your documentation with embedded models, code snippets, call outs and more to give your users the best experience.
```

File: netlify/landings/docs.yaml (hunk 8 above)
Comment (lottamus) [reply netlify/landings/docs.yaml]: I thought it got messed up in the merge. Feel free to change it back

Comment (marbemac) [reply netlify/landings/docs.yaml]: Yeah I actually like the simple almost blocky / low rez look of it - versus the image with tons of text (the actual docs page image).

**Pull request merged**